src/verifiers/documentStatus/didSignedDocumentStatus/didSignedDocumentStatus.ts

```diff
@@ -20,18 +23,152 @@ const skip: VerifierType["skip"] = async () => {
 };
 
 const test: VerifierType["test"] = (_document) => {
-  return true;
+  const document = _document as any; // TODO Casting to any first to prevent change at the OA level
+  if (document.proof && document.proof.some((proof: any) => proof.type === "OpenAttestationSignature2018")) return true;
+  return false;
 };
 
-const verify: VerifierType["verify"] = async (_document, _option) => {
+interface IdentityProof {
```

Comment:
all this should be at oa level

Reply:
not the point of this PR until we settle on the schema